mandatory/4-tax.js

```diff
@@ -17,7 +21,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(itemPrice) {
+  let totalPrice = calculateSalesTax(itemPrice);
+  let formatPrice = "£" + totalPrice.toFixed(2);
```

Comment:
Hi Laleh, good job with the tax homework. You can also use string interpolation in your formatPrice as another option. Well done!!